components/core/src/Grep.cpp

```diff
@@ -131,12 +134,12 @@ QueryToken::QueryToken (const string& query_string, const size_t begin_pos, cons
 
             if (!converts_to_non_dict_var) {
                 // Dictionary variable
-                m_type = Type::DictOrIntVar;
+                m_type = Type::DictionaryVar;
```

Comment:
This seems to be incorrect? what if a wildcard is at the middle?
example: log = "INFO static text 12345" and the query string is "12*45"